website/docs/03 - Schema/01 - Basics.md

```diff
@@ -62,7 +62,7 @@ For data located in a specific value of an attribute, the `path` will be a dot-d
 
 For data located in a specific subvalue of a value of an attribute, the `path` will be a dot-delimited string of integers with three parts. That is, the format of the string will be `"n.n.n"` where `n` represents an integer value. For instance, If the data is locatated in the first subvalue of the second value of the third attribute of a file, the `path` would be defined as `"3.2.1"`.
 
-:::caution
+:::warning[caution]
```

Comment:
It seems like the only difference between `:::warning[caution]` and `:::warning` is the title. I think its fine to let this be `:::warning`.

Reply:
Fixed.